backend/src/akvo/lumen/lib/auth.clj

```diff
@@ -0,0 +1,77 @@
+(ns akvo.lumen.lib.auth
+  (:require
+   [akvo.commons.jwt :as jwt]
+   [akvo.lumen.component.flow :as c.flow]
+   [akvo.lumen.component.tenant-manager :as tenant-manager]
+   [akvo.lumen.protocols :as p]
+   [clojure.spec.alpha :as s]
+   [clojure.string :as str]
+   [clojure.tools.logging :as log]
+   [hugsql.core :as hugsql]
+   [integrant.core :as ig]
+   [reitit.core :as rc]))
+
+(defrecord AuthorisedDBQueryService [tenant-conn authorised-uuid-tree]
+  p/DBQueryService
+  (p/query [this fun param-data options command-options]
```

Comment:
Why "fun" and not just "f"?

Reply:
fun is more funny :)

Reply:
and i have seen it in other code bases too